common/src/main/java/com/distkv/common/exception/PineCacheKeyNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.distkv.common.exception;
+
+
+public class PineCacheKeyNotFoundException extends DistkvException {
+
+
+  public PineCacheKeyNotFoundException(String errorMsg) {
+    super(errorMsg);
+  }
+
+
```

Comment:
Makea error code for **PineCacheKeyNotFoundException **.

Reply:
ok